createChannel.js

```diff
@@ -1,6 +1,6 @@
 const axios = require('axios');
 const {WebClient} = require('@slack/web-api');
-const config = require('./.config/.config.js');
+const config = require('./.config/development.js')
```

Comment:
Undo this please. Should be `require('./.config')`, and run with `env NODE_ENV=development` if you want to use development.js